### ISSUE TITLE
ReturnTypehintSpacingSniff - array return typehint is not checked

### ISSUE THREAD

**Issue by janlanger** [slevomat/coding-standard#65]
This method signature has whitespace between closing brace and return typehint colon. The sniff does not report this error.

```
public function createData(FooClass $foo) : array
```

Issue is limited only to array type, If the return typehint contains anything else, the error is reported.

**Comment by kukulich** [comment]
Added a workaround now: https://github.com/slevomat/coding-standard/commit/48e0da4a9562353faf380a062133564619516627
